### ISSUE TITLE
add premium seating

### ISSUE THREAD

**Issue by kevinfalting** [22arw/airshow#12]
We'll be selling "premium seating" so we need to make a `premiumseating` component so that we can drop it in on multiple pages.

The next thing we'll need is to find out how tickets will be sold. Do we need to build it or will it be sold through a third party website?

**Comment by kevinfalting** [comment]
I need a solid premium seating image with a filter over it.

**Comment by kevinfalting** [comment]
Tickets and payments will be managed on a third party website. TBD.

**Comment by kevinfalting** [comment]
Still waiting on information about how they want it implemented.

**Comment by kevinfalting** [comment]
Ticket sales managed at:
https://www.etix.com/ticket/v/13701/mcconnell-air-force-base?cobrand=FrontiersInFlight

**Comment by kevinfalting** [comment]
Closing this issue. When client wants to make changes, will open separate issue.
